exonum_client/client.py

```diff
@@ -69,6 +81,8 @@ def __exit__(self, exc_type: Optional[type], exc_value: Optional[Any], exc_trace
     def connect(self) -> None:
         """Connects the subscriber to the Exonum, so it will be able to receive events. """
         self._ws_client.connect(self._address)
+        if self._filters:
+            self._ws_client.send(json.dumps(self._filters))
```

Comment:
Hm. I thought that filters are provided via query string: [for example](https://github.com/exonum/exonum/blob/master/exonum/tests/websocket/main.rs#L191).

Have you checked if the implementation works?

Reply:
Indeed, it doesn't work properly. I tested it for "blocks" but forgot to test for "transactions".